### PR TITLE
If a Qt binding is already loaded, use it.

### LINKED ISSUE #87: Override QT_API when a Qt binding is already loaded?
I would like to suggest not obeying QT_API when a specific Qt binding has already been loaded, and just use it, rather than raising an exception: the error is probably just a misconfigured environment variable and there's a clear, single, well-defined recovery.

### PR DESCRIPTION
**Pull request by anntzer**
Closes #87.

### CODE REVIEW

File: qtconsole/qt_loaders.py [hunk 1]
```diff
@@ -240,30 +240,37 @@ def import_pyside():
...
 
     Raises
     ------
-    ImportError, if it isn't possible to import any requested
-    bindings (either becaues they aren't installed, or because
-    an incompatible library has already been installed)
+    ImportError, if it isn't possible to import any requested bindings
+    (because they aren't installed).
     """
+    if "PySide" in sys.modules:
```
Comment (takluyver) [inline qtconsole/qt_loaders.py]: This should use the `loaded_api()` check that already exists in the module. It's possible, if unusual, to load the top-level namespace of one of these packages without actually loading the bindings:

```
In [1]: import PyQt4

In [2]: import PyQt5.QtCore

In [3]: 
```

`loaded_api()` checks whether `QtCore` is loaded.

Comment (anntzer) [inline qtconsole/qt_loaders.py]: fixed.

**Comment by takluyver** [comment]
Thanks, looks good to me. I'll give other people a day or so to look at it, then merge.

**Comment by minrk** [comment]
:+1: @ccordoba12?

**Comment by ccordoba12** [comment]
:+1: from me too

**Pull request merged**